tools/offsets-tool-py/offsets-tool.py

```diff
@@ -84,6 +87,33 @@ def require_emscipten_path (args):
 			self.target = Target ("TARGET_WASM", None, [])
 			self.target_args += ["-target", args.abi]
 
+		# Linux
+		elif "arm-linux-gnueabihf" == args.abi:
```

Comment:
can we add `arm-linux-gnueabi` and  `aarch64-linux-gnu` too? this is awesome, thanks ❤️

Reply:
I would suggest that we first get `arm-linux-gnueabihf` working on CI and then add the other targets. I'm sure there are bits missing :)